src/fp3.rs

```diff
@@ -260,6 +299,13 @@ mod tests {
         }
     }
 
+    // DISPLAY
+    // ================================================================================================
+    #[test]
+    fn test_debug() {
+        assert_eq!(format!("{:?}", Fp3::zero()), "0");
```

Comment:
Nit, but coefficients are crate-accessible, so you could have still tested the various cases you had originally, by directly calling say
```rust
Fp3 {
    a0: Fp::one(),
    a1: Fp::new(3),
    a2: Fp::new(5),
}
```
but again that's not a big deal as this is not part of the API.